include_all_in_one/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# This script will create the amalgamated library into 
+#    include_all_in_one/include/fplus/fplus.hpp
+#
+# Notes:
+# - include_all_in_one/include/fplus/fplus.hpp is part of the source repository, so that using FunctionalPlus 
+#   online with tools such as Compiler Explorer is easy. Just write:
+#     #include "https://raw.githubusercontent.com/Dobiasd/FunctionalPlus/master/include_all_in_one/include/fplus/fplus.hpp"
+# - this file will be updated automatically at each build, so that library developpers do not forget 
+#   to commit it (ideally once per PR)
+find_package (Python COMPONENTS Interpreter QUIET)
+if (Python_FOUND)
+    add_custom_target(
+        include_all_in_one
+        ALL
+        COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_LIST_DIR}/make_all_in_one.py
+    )
+else()
+    message(STATUS "Not generating include_all_in_one, python was not found.")
```

Comment:
The message isn't quite right. This command adds a target that when built generates the all in one, it doesn't generate it by itself. The more correct message would be `Python not found, not adding the 'include_all_in_one' target`